presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestInsertIntoHiveTable.java

```diff
@@ -56,35 +54,25 @@ public Requirement getRequirements(Configuration configuration)
         }
     }
 
-    private static class AllSimpleTypesKnownToPrestoTables
-            implements RequirementsProvider
-    {
-        @Override
-        public Requirement getRequirements(Configuration configuration)
-        {
-            return compose(
-                    mutableTable(ALL_HIVE_SIMPLE_TYPES_KNOWN_TO_PRESTO_TEXTFILE, TABLE_NAME, CREATED),
-                    immutableTable(ALL_HIVE_SIMPLE_TYPES_KNOWN_TO_PRESTO_TEXTFILE));
-        }
-    }
-
-    @Test(groups = {HIVE_CONNECTOR, QUARANTINE})
+    @Test(groups = {HIVE_CONNECTOR, SMOKE})
     @Requires(AllSimpleTypesTables.class)
     public void testInsertIntoValuesToHiveTableAllHiveSimpleTypes()
     {
         String tableNameInDatabase = mutableTablesState().get(TABLE_NAME).getNameInDatabase();
         assertThat(query("SELECT * FROM " + tableNameInDatabase)).hasNoRows();
         query("INSERT INTO " + tableNameInDatabase + " VALUES(" +
-                "127, " +
-                "32767, " +
+                "TINYINT '127', " +
+                "SMALLINT '32767', " +
                 "2147483647, " +
                 "9223372036854775807, " +
-                "123.345, " +
+                "FLOAT '123.345', " +
                 "234.567, " +
+                "cast (345.678 as DECIMAL(10, 0)), " +
+                "cast (345.678 as DECIMAL(10, 5))," +
                 "timestamp '2015-05-10 12:15:35.123', " +
                 "date '2015-05-10', " +
                 "'ala ma kota', " +
-                "'ala ma kota', " +
+                "cast ('ala ma kota' as VARCHAR(10)), " +
```

Comment:
I didn't change the type in the DDL, the semantics for VARCHAR (in particular parameterized varchar) must have changed since this test was written.